products/sle/main.pm

```diff
@@ -990,8 +990,12 @@ elsif (get_var("VIRT_AUTOTEST")) {
     load_boot_tests();
     load_inst_tests();
     loadtest "virt_autotest/login_console.pm";
+    if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen")) {
+        loadtest "virt_autotest/setup_console_on_host.pm";
+        loadtest "virt_autotest/reboot_and_wait_up_normal1.pm";
+    }
     loadtest "virt_autotest/install_package.pm";
-    loadtest "virt_autotest/reboot_and_wait_up_normal1.pm";
+    loadtest "virt_autotest/reboot_and_wait_up_normal2.pm";
```

Comment:
what do these numbers in reboot_and_wait_up_normal1-3 mean? Shouldn't the name be improved?